test/test_api_version.py

```diff
@@ -32,6 +32,7 @@ def test_normalize_version(self):
 
     def test_is_supported_version(self):
         self.assertTrue(api_version.is_supported_version("2.0"), "2.0 is a supported version")
+        self.assertTrue(api_version.is_supported_version("2.1"), "2.1 is a supported version")
```

Comment:
Having a separate `is_slightly_future_version()` method would allow us to not formalize support for a version that doesn't exist yet. Future proofing APIs is hard :)